pandas/core/apply.py

```diff
@@ -482,7 +482,7 @@ def wrap_results_for_axis(
             return res
 
         elif self.result_type is None and all(
-            isinstance(x, dict) for x in results.values()
+            isinstance(x, dict) or isinstance(x, tuple) for x in results.values()
```

Comment:
can you use
```isinstance(x, (dict, tuple))```

Reply:
actually you try using `is_list_like(x)`, its possible this breaks other things and you may have to do `is_list_like(x) and not isinstance(x, Series)`

Reply:
Yes, it breaks a few tests.. I tried it already. :)